src/Transformer.php

```diff
@@ -39,5 +45,15 @@ public function prepareTransformation()
         return $merged->parseNested();
     }
 
+    public function getOptions(): array
+    {
+        return $this->options;
+    }
+
+    protected function with(string $transformerClass, $paths = null): Transformer
+    {
+        return new $transformerClass($this->get($paths), $paths, $this->options);
```

Comment:
Use the method `$this->options()` here.

Reply:
I usually use the properties in an object. But I can use the method in this case.